BrowserKit/Sources/ToolbarKit/AddressToolbar/LocationView/LocationView.swift

```diff
@@ -47,6 +49,8 @@ final class LocationView: UIView, LocationTextFieldDelegate, ThemeApplicable, Ac
 
     private lazy var urlTextFieldColor: UIColor = .black
     private lazy var urlTextFieldSubdomainColor: UIColor = .clear
+    private lazy var lockIconImageColor: UIColor = .clear
+    private lazy var safeListedURLImageColor: UIColor = .clear
```

Comment:
Total nit (feel free to ignore): My brain freaks out because the order of `lockIconImageColor` and `safeListedURLImageColor` is different here compared to `safeListedURLImageName` and `lockIconImageName` a few lines up 🙈